### ISSUE TITLE
NotFoundError (see above for traceback): Failed to create a NewWriteableFile:ϵͳ\udcd5Ҳ\udcbb\udcb5\udcbdָ\udcb6\udca8\udcb5\udcc4·\udcbe\udcb6\udca1\udca3

### ISSUE THREAD

**Issue by KangCai** [KangCai/bugwall#1]
> Caused by op 'save/SaveV2', defined at:
  File "D:/game/G78/z_intern_client_game/experimental/beyond/data_preparation/runtime_winning_prediction/wd_model.py", line 137, in <module>
    train()
  File "D:/game/G78/z_intern_client_game/experimental/beyond/data_preparation/runtime_winning_prediction/wd_model.py", line 125, in train
    model.train(train_files)
  File "D:/game/G78/z_intern_client_game/experimental/beyond/data_preparation/runtime_winning_prediction/wd_model.py", line 56, in train
    self.estimator.train(input_fn=lambda: input_fn(tfrecords_filename, True), steps=10000)
  File "C:\Users\caikang\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\estimator\estimator.py", line 355, in train
    loss = self._train_model(input_fn, hooks, saving_listeners)
  File "C:\Users\caikang\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\estimator\estimator.py", line 900, in _train_model
    log_step_count_steps=self._config.log_step_count_steps) as mon_sess:
  File "C:\Users\caikang\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\training\monitored_session.py", line 384, in MonitoredTrainingSession
    stop_grace_period_secs=stop_grace_period_secs)
  File "C:\Users\caikang\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\training\monitored_session.py", line 795, in __init__
    stop_grace_period_secs=stop_grace_period_secs)
  File "C:\Users\caikang\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\training\monitored_session.py", line 518, in __init__
    self._sess = _RecoverableSession(self._coordinated_creator)
  File "C:\Users\caikang\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\training\monitored_session.py", line 981, in __init__
    _WrappedSession.__init__(self, self._create_session())
  File "C:\Users\caikang\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\training\monitored_session.py", line 986, in _create_session
    return self._sess_creator.create_session()
  File "C:\Users\caikang\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\training\monitored_session.py", line 675, in create_session
    self.tf_sess = self._session_creator.create_session()
  File "C:\Users\caikang\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\training\monitored_session.py", line 437, in create_session
    self._scaffold.finalize()
  File "C:\Users\caikang\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\training\monitored_session.py", line 214, in finalize
    self._saver.build()
  File "C:\Users\caikang\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\training\saver.py", line 1320, in build
    self._build(self._filename, build_save=True, build_restore=True)
  File "C:\Users\caikang\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\training\saver.py", line 1357, in _build
    build_save=build_save, build_restore=build_restore)
  File "C:\Users\caikang\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\training\saver.py", line 800, in _build_internal
    save_tensor = self._AddShardedSaveOps(filename_tensor, per_device)
  File "C:\Users\caikang\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\training\saver.py", line 411, in _AddShardedSaveOps
    return self._AddShardedSaveOpsForV2(filename_tensor, per_device)
  File "C:\Users\caikang\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\training\saver.py", line 385, in _AddShardedSaveOpsForV2
    sharded_saves.append(self._AddSaveOps(sharded_filename, saveables))
  File "C:\Users\caikang\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\training\saver.py", line 326, in _AddSaveOps
    save = self.save_op(filename_tensor, saveables)
  File "C:\Users\caikang\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\training\saver.py", line 242, in save_op
    tensors)
  File "C:\Users\caikang\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\ops\gen_io_ops.py", line 1793, in save_v2
    shape_and_slices=shape_and_slices, tensors=tensors, name=name)
  File "C:\Users\caikang\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\framework\op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "C:\Users\caikang\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\framework\ops.py", line 3290, in create_op
    op_def=op_def)
  File "C:\Users\caikang\AppData\Local\Programs\Python\Python35\lib\site-packages\tensorflow\python\framework\ops.py", line 1654, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

> NotFoundError (see above for traceback): Failed to create a NewWriteableFile: ../data/both/models_list_wide_n_deep/2/model.ckpt-10000_temp_15fe9c5263f64f68afef738ce733f441/part-00000-of-00001.data-00000-of-00001.tempstate12204665382205475603 : ϵͳ\udcd5Ҳ\udcbb\udcb5\udcbdָ\udcb6\udca8\udcb5\udcc4·\udcbe\udcb6\udca1\udca3
; No such process
